deploy/helms/legion/templates/edi/edi-secrets.yaml

```diff
@@ -29,5 +29,14 @@ type: Opaque
 data:
   grafana.user: YWRtaW4=
   grafana.password: YWRtaW4=
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Release.Name }}-jwt-config"
+type: Opaque
+data:
+  jwt.secret: "{{ .Values.secrets.token_secret }}"
```

Comment:
Why do you read ` .Values.secrets.token_secret` but write `secrets.tokenSecret`?